### PR TITLE
fix package

### PR DESCRIPTION
**Pull request by graingert**
Hey there, welcome to Python!

Here we use reStructuredText rather than markdown because it renders as HTML on PyPI.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,26 +4,24 @@
...
 
-VERSION = '0.1.6'
 
-long_description = '''torch-vision provides DataLoaders, Pre-trained models
-and common transforms for torch for images and videos'''
+readme = open('README.rst').read()
+
+VERSION = '0.1.6'
 
-setup_info = dict(
```
Comment (graingert) [inline setup.py]: no need to make a dict then unpack it.

Comment (graingert) [inline setup.py]: also best to make dicts with a literal `{'name': 'torchvision'}`

**Comment by soumith** [comment]
hi, the README looks much worse after this change on github. Is there a way to keep it as markdown? Maybe we can use an on the fly md -> rst converter?

screen shot 2017-01-18 at 6 44 32 pm

**Comment by graingert** [comment]
@soumith hmm github should be rendering it... these are the problems with on-the-fly conversion.

**Comment by graingert** [comment]
@soumith if you `pip install docutils` and run `python3 setup.py check -r -s` you'll be able to see what's up. It looks like pandoc has converted some of the headings incorrectly.

**Comment by graingert** [comment]
@soumith it looks like:
```rst
ham
~~~
```
isn't valid rst :( do you want to pick a different one?

**Comment by graingert** [comment]
@soumith ok that's the readme fixed.

**Comment by graingert** [comment]
@soumith here's what it looks like on PyPI https://pypi.org/project/torchvision/0.1.6.dev0/

**Comment by soumith** [comment]
this is awesome! thank you so much for fixing the readme formatting. 

Did you already upload it to pypi, or do I have to do anything further?

**Comment by graingert** [comment]
The main torch repo will need the same treatment. It should be easy with pandoc and docutils and some fiddling about

**Comment by graingert** [comment]
Yeah I'll push it to PyPI in the morning

**Comment by soumith** [comment]
I need to also build a proper wheel for the main torch repo. Right now it's a bit of a hack via: https://github.com/pytorch/builder/blob/master/wheel/build_wheel.sh

Your restructuring on this repo helped me understand what needs to be done. Thanks.

In the future when new releases happen, can I upload to PyPI? I'm going to sit and read the docs for that in the next couple of days.

**Comment by graingert** [comment]
You need to fix this so it generates manylinux1 wheels

**Comment by graingert** [comment]
Otherwise it's not portable

**Pull request merged**